.travis.yml

```diff
@@ -4,4 +4,3 @@ scala:
   - 2.11.7
 jdk:
   - oraclejdk7
-  - openjdk7
```

Comment:
I think would be better to add ticket for bringing it back since its anyway doesn't work for now? https://github.com/snowplow/schema-ddl/issues/47